react_app/src/components/CoursePage/CourseOverview.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import "./CoursePage.css"
+import {Box, Typography} from '@material-ui/core';
+import Rating from './Rating';
+import RateCourseButton from './RateCourseButton';
+import {connect} from 'react-redux';
+import {fetchCourseInfo, fetchReviews, toggleAddReviewBox} from '../../actions';
+
+class CourseOverview extends React.Component {
+
+    constructor(props) {
+        super(props);
```

Comment:
I get a warning that says Useless Contructor

Reply:
oh yea youre right lemme get rid of that